src/test/java/com/puppycrawl/tools/checkstyle/internal/AllChecksTest.java

```diff
@@ -243,30 +241,15 @@ public void testAllChecksWithDefaultConfiguration() throws Exception {
         final String[] expected = CommonUtils.EMPTY_STRING_ARRAY;
 
         for (Class<?> check : CheckUtil.getCheckstyleChecks()) {
-            final DefaultConfiguration checkConfig = createCheckConfig(check);
+            final DefaultConfiguration checkConfig = createModuleConfig(check);
             final Checker checker;
-            if (AbstractCheck.class.isAssignableFrom(check)) {
```

Comment:
@romani This removal is because the new `createChecker` method will handle checks as well as filesets.

However, this test is slightly broken. Even though it checks for AbstractChecks versus FileSets, `CheckUtil.getCheckstyleChecks` only returns AbstractChecks, so we are ignoring all FileSets.
Once this PR is merged I will look at fixing this test.